### PR TITLE
11436 Added Expandable Help component, story, test suite

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* bcgov/entity#11436

*Description of changes:*
- rebuilt main package-lock.json
- added Expandable Help component, story, test suite
- updated Storybook

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: docs/assets/ExpandableHelp.stories-12517aed.js
Comment (severinbeauvais) [inline docs/assets/ExpandableHelp.stories-12517aed.js:1]: You can ignore all files in docs/ as they are auto-generated StoryBook files. Or you can review just the first commit (as the SB updates are in the second commit).

File: package-lock.json
Comment (severinbeauvais) [inline package-lock.json:1]: I deleted and rebuilt this file, which is good to do from time to time.

File: src/components/expandable-help/ExpandableHelp.stories.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    <expandable-help v-bind="$props">
+      <template #content>${args.content}</template>
+    </expandable-help`,
+  methods: {}
+})
+
+export const base = Template.bind({})
+base['args'] = {
+  helpLabel: 'Show Sample Help',
+  content: `sample help text`
```
Comment (severinbeauvais) [inline src/components/expandable-help/ExpandableHelp.stories.ts:23]: The template content can be passed in as an arg, but it's handled explicitly on line 15.

File: src/components/expandable-help/ExpandableHelp.vue
Comment (severinbeauvais) [inline src/components/expandable-help/ExpandableHelp.vue:1]: This is the same component as in auth-web (with a bit of layout cleanup).

**Comment by severinbeauvais** [comment]
In Storybook:

image

File: src/components/expandable-help/package.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  "name": "@bcrs-shared-components/expandable-help",
+  "version": "0.0.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "vue": "^2.7.14"
+  },
+  "devDependencies": {
+    "vue-property-decorator": "^9.1.2"
```
Comment (JazzarKarim) [inline src/components/expandable-help/package.json:11]: One of the things that we're going to have to get rid of in this repo 😟 . Here and in all the other components.

**Review by JazzarKarim — APPROVED** [review]
Looks great Sev! 👍

File: src/components/expandable-help/package.json (hunk 2 above)
Comment (severinbeauvais) [reply src/components/expandable-help/package.json:11]: Yup. We can do it all together (future).

**Review by PaulGarewal — APPROVED** [review]
It's good to see how changes in this repo look. 

Looks great to me, thanks for involving us Sev

**Review by chenhongjing — APPROVED** [review]
LGTM 😺

**Pull request merged**